communications/newsletters/newsletter_31_Jan2022.md

```diff
@@ -0,0 +1,156 @@
+He is a certified Software Carpentry instructor and is frequently involved in organising training events.
+As a member of *The Turing Way* core team, he supports researchers from eScience Center in sharing internally developed resources and recommended practices for the [Netherlands eScience Center guide](https://guide.esciencecenter.nl/#/) to *The Turing Way*.
+
+![Carlos Martinez's headshot](images/2022-01-carlos-4.png)
+
+In his words:
+> We always advocate for software reuse and collaborative development of software. 
+> I love that we can do the same for software development guidelines: reuse content from the eScience guide and collaboratively develop with The Turing Way community!
+> I am a big advocate of improving software quality. 
+> I am really glad that the eScience center is collaborating with The Turing Way in providing guidelines and helping build better research software.
+
+Thanks to Carlos’ work at organisation level.
+We are using this example to **encourage documentation of internal practises** from different research teams and cross-reference them to *The Turing Way* **enabling community-led support and sustainability** for their work.
+
+### New Chapters
+
+* Arielle Bennett, Esther Plomp, Emma Karoune, Vicky Hellon and Lena Karvovskaya collaborated to document [Research Infrastructure Roles](https://the-turing-way.netlify.app/collaboration/research-infrastructure-roles.html) that includes subchapters on Community Managers, Data Stewards and Research Software Engineers.
+* Nina di Cara published a chapter on [self-reflection guidance for ethical research](https://the-turing-way.netlify.app/ethical-research/self-reflection.html).
+* Faruk D., Abel Siqiera and Carlos Martinez worked on [Software Citation with CITATION.cff](https://the-turing-way.netlify.app/communication/citable/citable-cff.html) and using [CFFLint](https://the-turing-way.netlify.app/communication/citable/citable-cffinit.html).
+* Margaret Wanjiku added the [documentation and metadata](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-metadata.html) subchapter in the Research Data Management chapter.
+* Vicky Hellon revised and updated the [Open Access](https://the-turing-way.netlify.app/reproducible-research/open/open-access.html) publication subchapter.
+* Malvika ana Alejandro Coca added a simple guidance for chairing [Collaboration Café](https://the-turing-way.netlify.app/community-handbook/coworking/coworking-collabcafe.html#chairing-an-online-collaboration-cafe) and [Coworking Calls](https://the-turing-way.netlify.app/community-handbook/coworking/coworking-weekly.html#chairing-coworking-calls).
+
+*Have an idea for collaborations or a chapter? [Connect with us](https://github.com/alan-turing-institute/the-turing-way/blob/main/CONTRIBUTING.md#get-in-touch). Please explore [issues]() and Pull Requests](https://github.com/alan-turing-institute/the-turing-way/pulls) to collaborate with others on ongoing work.*
```

Comment:
Missing link for issues: https://github.com/alan-turing-institute/the-turing-way/issues

Reply:
```suggestion
*Have an idea for collaborations or a chapter? [Connect with us](https://github.com/alan-turing-institute/the-turing-way/blob/main/CONTRIBUTING.md#get-in-touch). Please explore [issues](https://github.com/alan-turing-institute/the-turing-way/issues) and Pull Requests](https://github.com/alan-turing-institute/the-turing-way/pulls) to collaborate with others on ongoing work.*
```